src/datta_lab_to_nwb/markowitz_gillis_nature_2023/fiberphotometryinterface.py

```diff
@@ -0,0 +1,220 @@
+                excitation_sources=excitation_sources_table,
+                photodetectors=photodetectors_table,
+                fluorophores=fluorophores_table,
+            )
+        )
+
+        # Important: we add the fibers to the fibers table _after_ adding the metadata
+        # This ensures that we can find this data in their tables of origin
+        fibers_table.add_fiber(
+            excitation_source=0,  # integers indicated rows of excitation sources table
+            photodetector=0,
+            fluorophores=[0],  # potentially multiple fluorophores, so list of indices
+            location=metadata["FiberPhotometry"]["area"],
+            notes="None",
+        )
+        fibers_table.add_fiber(
+            excitation_source=1,  # integers indicated rows of excitation sources table
+            photodetector=0,
+            fluorophores=[0],  # potentially multiple fluorophores, so list of indices
+            location=metadata["FiberPhotometry"]["area"],
+            notes="None",
+        )
+
+        # ROI Response Series
+        # Here we set up a list of fibers that our recording came from
+        fibers_ref = DynamicTableRegion(name="rois", data=[0, 1], description="source fibers", table=fibers_table)
+        signal_series = RoiResponseSeries(
+            name="SignalDfOverF",
+            description="The ΔF/F from the blue light excitation (480nm) corresponding to the dopamine signal.",
+            data=H5DataIO(session_df.signal_dff.to_numpy(), compression=True),
+            unit="a.u.",
+            timestamps=H5DataIO(session_df.timestamp.to_numpy(), compression=True),
+            rois=fibers_ref,
+        )
+        reference_series = RoiResponseSeries(
+            name="ReferenceDfOverF",
+            description="The ∆F/F from the isosbestic UV excitation (400nm) corresponding to the reference signal.",
+            data=H5DataIO(session_df.reference_dff.to_numpy(), compression=True),
+            unit="a.u.",
+            timestamps=signal_series.timestamps,
+            rois=fibers_ref,
+        )
+        reference_fit_series = RoiResponseSeries(
+            name="ReferenceDfOverFSmoothed",
+            description=(
+                "The ∆F/F from the isosbestic UV excitation (400nm) that has been smoothed "
+                "(See Methods: Photometry Active Referencing)."
+            ),
+            data=H5DataIO(session_df.reference_dff_fit.to_numpy(), compression=True),
+            unit="dF/F",
+            timestamps=signal_series.timestamps,
+            rois=fibers_ref,
+        )
+        uv_reference_fit_series = RoiResponseSeries(
+            name="UVReferenceFSmoothed",
+            description=(
+                "Raw F from the isosbestic UV excitation (400nm) that has been smoothed "
+                "(See Methods: Photometry Active Referencing)."
+            ),
+            data=H5DataIO(session_df.uv_reference_fit.to_numpy(), compression=True),
+            unit="F",
```

Comment:
```suggestion
            unit="n.a.",
```

This is the case where they are not actually unitless, but I assume we don't have the exact luminosity in scientific units